<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>611</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>729</width>
    <height>611</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_log">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="image_resource.qrc">:/new/prefix1/image_resource/team_log4.jpg</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="team_name_label">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>21</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 70 12pt &quot;Courier 10 Pitch&quot;;
color: rgb(0, 170, 255);</string>
           </property>
           <property name="text">
            <string>TEAM EXPLORER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>111</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 italic 11pt &quot;AR PL UMing TW MBE&quot;;
color: rgb(210, 32, 32);</string>
           </property>
           <property name="text">
            <string>urban search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>111</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 italic 11pt &quot;AR PL UMing TW MBE&quot;;
color: rgb(45, 80, 47);</string>
           </property>
           <property name="text">
            <string>rescue</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber_time">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>91</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(34, 27, 27);
color: rgb(211, 26, 19);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_database">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;数据库操作&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="image_resource.qrc">
             <normaloff>:/new/prefix1/image_resource/database.png</normaloff>:/new/prefix1/image_resource/database.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>81</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>数据库操作</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_user">
           <property name="maximumSize">
            <size>
             <width>81</width>
             <height>71</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;管理员登陆安全&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="image_resource.qrc">
             <normaloff>:/new/prefix1/image_resource/refresh.png</normaloff>:/new/prefix1/image_resource/refresh.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>       刷新</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_close">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;退出系统&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="image_resource.qrc">
             <normaloff>:/new/prefix1/image_resource/close.png</normaloff>:/new/prefix1/image_resource/close.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>53</width>
             <height>60</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>退出系统</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="treeView_classroom">
      <property name="minimumSize">
       <size>
        <width>201</width>
        <height>481</height>
       </size>
      </property>
     </widget>
     <widget class="ImageFrame" name="frame_image">
      <property name="minimumSize">
       <size>
        <width>471</width>
        <height>481</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageFrame</class>
   <extends>QFrame</extends>
   <header location="global">image_frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image_resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>756</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
